src/lib/snyk/displayOutput.ts

```diff
@@ -1,182 +1,183 @@
 import * as _ from 'lodash';
```

Comment:
Why is this file changed? is it just formatting?

Reply:
tep second commit is formatting only from prettier https://github.com/snyk-tech-services/snyk-delta/pull/140/commits/9764245ea4bb4721df781ef239634fff4dbad6ee